backend/src/services/artist.service.ts

```diff
@@ -1,33 +1,75 @@
 import { Artist, ArtistInterface } from "../models/artist.model";
 import { UpdateOptions } from "sequelize";
+import { RED } from "../config/constants";
 
 export class ArtistService {
-  public async create(artist: ArtistInterface) {
-    const createdArtist: Artist = await Artist.create<Artist>(artist);
-    return { success: true, body: createdArtist };
+  /**
+   * Creates a new artist
+   * @param artist ArtistInterface describing newly created artist
+   */
+  public async create(artist: ArtistInterface): Promise<Artist> {
+    try {
+      const createdArtist: Artist = await Artist.create<Artist>(artist);
+      return createdArtist;
+    } catch (e) {
+      console.error(RED, e);
+      throw e;
+    }
   }
 
-  public async show(artistId: number) {
-    const artist = await Artist.findByPk<Artist>(artistId, {
-      rejectOnEmpty: true,
-    });
-    return { success: true, artist: artist };
+  /**
+   * show an artist by id
+   * @param artistId id of artist to show
+   */
+  public async show(artistId: number): Promise<Artist> {
+    try {
+      const artist: Artist = await Artist.findByPk<Artist>(artistId, {
+        rejectOnEmpty: true,
+      });
+      return artist;
```

Comment:
will returning `artist` directly rather than `{artist:artist}` require refactoring the muralForm in the frontend?

Reply:
I did refactor it, only 1 component uses this endpoint as far as i know